cmd/generate/config/rules/azure.go

```diff
@@ -0,0 +1,458 @@
+package rules
+
+import (
+	"github.com/zricethezav/gitleaks/v8/cmd/generate/secrets"
+	"github.com/zricethezav/gitleaks/v8/config"
+)
+
+// Rules come from https://www.powershellgallery.com/packages/AzSK.AzureDevOps/0.9.8/Content/Framework%5CConfigurations%5CSVT%5CAzureDevOps%5CCredentialPatterns.xml
+// Only rules with 'ContentSearchPatterns' have been used.
+
+// CSCAN0110, CSCAN0111, CSCAN0140, CSCAN0220 searches for generic passwords - covered elsewhere
+
+// CSCAN0120 searches for Twilio keys - covered in twilio.go
+
+// CSCAN0210 checks for Git repo credentials - covered elsewhere
+
+// CSCAN0230 checks for Slack tokens - covered in slack.go
+
+// CSCAN0250 - covered in jwt.go
+
+func AzureAppServiceDeploymentSecrets() *config.Rule {
+	// define rule
+	r := config.Rule{
+		Description: "CSCAN0020, CSCAN0030 - Found Azure app service deployment secrets in publish settings file.",
+		RuleID:      "azure-app-service-deployment-secrets",
+		SecretGroup: 1,
+		Regex:       generateUniqueTokenRegex(`MII[a-z0-9=_\-]{200}`, true),
+		Keywords:    []string{"MII"},
+	}
+
+	// validate
+	tps := []string{
+		generateSampleSecret("azure-app-service-deployment-secrets",
+			"MII"+secrets.NewSecret(alphaNumeric("200"))),
+	}
+	return validate(r, tps, nil)
+}
+
+func AzureStorageCredential86char() *config.Rule {
+	// define rule
+	r := config.Rule{
+		Description: "CSCAN0030, CSCAN0090, CSCAN0150 - Found Azure storage credential in source code file.",
+		RuleID:      "azure-storage-credential-86char",
+		SecretGroup: 1,
+		Regex:       generateUniqueTokenRegex(`[ \t]{0,10}[a-zA-Z0-9/+]{86}==`, true),
+		Keywords:    []string{"=="},
+	}
+
+	// validate
+	tps := []string{
+		generateSampleSecret("azure-storage-credential-86char",
+			secrets.NewSecret(alphaNumeric("86")+"==")),
+	}
+	return validate(r, tps, nil)
+}
+
+func AzureStorageCredential43char() *config.Rule {
```

Comment:
I've tested both these out and they're far too noisy as-is. I'm not sure how they can be improved beyond adding a prefix for "accountkey", though that may overlap with an existing pattern.